### PR TITLE
Expose public Schema and WriteChunk on ArrowWriter

### PR DESCRIPTION
**Pull request by akotlar**
* Expose public Schema and WriteChunk on ArrowWriter
* Add NewIPCFileWriterWithSchema to make it possible to pass schema from existent IPC file

### CODE REVIEW

File: arrow/arrow.go [hunk 1]
```diff
@@ -22,21 +22,23 @@ type ArrowWriter struct {
...
 
 	return &ArrowWriter{
-		schema: schema,
+		Schema: schema,
 		writer: writer,
 	}, nil
 }
 
-func (aw *ArrowWriter) writeChunk(record arrow.Record) error {
+func (aw *ArrowWriter) WriteChunk(record arrow.Record) error {
```
Comment (cristinaetrv) [inline arrow/arrow.go:41]: It still says writeChunk on line 110 and 121

Comment (akotlar) [reply arrow/arrow.go:41]: Ah those are methods on ArrowRowBuilder, which does have a writeChunk method.

**Comment by akotlar** [comment]
Addressed comments

**Review by cristinaetrv — APPROVED** [review]
Lgtm!

**Pull request merged**